api/v1/controllers/UserController.js

```diff
@@ -90,7 +90,8 @@ function requestPasswordReset(req, res, next) {
 router.use(bodyParser.json());
 router.use(middleware.auth);
 
-router.post('/', middleware.request(requests.BasicAuthRequest), createUser);
+router.post('/', middleware.request(requests.BasicAuthRequest),
+  middleware.permission(roles.ORGANIZERS), createUser);
```

Comment:
Is this really what we want? I'd hate to lose people who don't have a GitHub/don't know what a GitHub is by forcing them to authenticate with one.